packages/backend-data/API.md

```diff
@@ -30,6 +30,7 @@ export type DataProps = {
     name?: string;
     authorizationModes?: AuthorizationModes;
     functions?: Record<string, ConstructFactory<AmplifyFunction>>;
+    experimentalStackMapping?: Record<string, string>;

```

Comment:
Let's review the API design internally. We may need a consolidated attribute to configure experimental features.

Reply:
It's just works like gen1. It was working property on Gen 1. Now it can be usable on Gen 2. I don't know if it's experimental or not for this reason but it's works on Cloudformation same as gen 1.

But changing stacks with gen 1 or gen 2 not effecting resource limit on Cloudformation. These are still hits the resource limit